docs/_data/docstoc.yml

```diff
@@ -45,6 +45,8 @@
         url: che-setupregistries.html
       - title: 'Creating a Codewind workspace in Che'
         url: che-createcodewindworkspace.html
+      - title: Making a code change with Codewind for Eclipse Che
```

Comment:
title: missing the quotes (' ') so change to 'Making a code change with Codewind for Eclipse Che' otherwise lgtm.